src/ConfigProvider.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Mvc\View;
+
+use Laminas\ServiceManager\ConfigInterface;
+use Laminas\View\Helper as ViewHelper;
+
+/** @psalm-import-type ServiceManagerConfigurationType from ConfigInterface */
+final class ConfigProvider
+{
+    /** @return array<string, mixed> */
+    public function __invoke(): array
+    {
+        return [
+            'dependencies' => $this->getDependencyConfig(),
+            'view_helpers' => $this->getViewHelperConfig(),
+            /**
+             * Configuration for Individual View Helpers
+             */
+            'view_helper_config' => [
```

Comment:
Can we create psalm types for configuration consumed by this model? Then we could use that in consumers.